### PR TITLE
E86 Added TimeValue type

### PR DESCRIPTION
**Pull request by alexisKinto**
- Added Type: `TimeValue` and its corresponding tests.
- Fixed importing dummy environment file for rails helper

Note: To make use of this TimeValue Type make sure to include it in your app:
```
# config/initializers/bali_view_components.rb

ActiveSupport.on_load :active_record do
  include Bali::Types
end
```

Closes: https://github.com/Grupo-AFAL/bali-view-components/issues/86

### CODE REVIEW

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -14,9 +14,10 @@
   add_filter(/preview.rb/)
 end
 
+require File.expand_path('./dummy/config/environment', __dir__)
```
Comment (fedegl) [inline spec/rails_helper.rb:17]: porque es necesario cargar antes del spec_helper?

File: spec/types/time_value_spec.rb [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    end
+
+    it 'serializes a date string to a number in seconds' do
+      workout = Workout.create(workout_start_at: "#{Date.current} 01:00:00")
+      stored_value = workout.read_attribute_before_type_cast(:workout_start_at)
+      expect(stored_value).to eql(3600)
+    end
+  end
+
+  subject { Bali::Types::TimeValue.new }
```
Comment (rafaelcdelgado) [inline spec/types/time_value_spec.rb:17]: This can be used as `described_class.new` , isnt?

File: spec/types/time_value_spec.rb [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+  describe '#cast' do
+    it 'return a default time if value is blanck' do
+      expect(subject.cast('')).to eql("#{Date.current} 00:00:00")
+    end
+
+    it 'returns value if is a date string' do
+      date_string = "#{Date.current} 00:00:00"
```
Comment (rafaelcdelgado) [inline spec/types/time_value_spec.rb:25]:
```suggestion
      date_string = "#{Date.current} 00:00:00"
```
This is a DateTime string, not date.

**Review by fedegl — APPROVED** [review]

File: spec/dummy/app/models/workout.rb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class Workout < ApplicationRecord
+  attribute :workout_start_at, TimeValue.new
```
[the remark this answers is not in the document]
Comment (rafaelcdelgado) [reply spec/dummy/app/models/workout.rb:4]: Make more concise..

File: spec/dummy/db/migrate/20210729212951_create_workouts.rb [hunk 5]
```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class CreateWorkouts < ActiveRecord::Migration[6.1]
+  def change
+    create_table :workouts do |t|
+      t.integer :workout_start_at
```
Comment (rafaelcdelgado) [inline spec/dummy/db/migrate/20210729212951_create_workouts.rb:6]:
```suggestion
      t.integer :start_at
```
same here

**Pull request merged**